pkg/network/dns_cache.go

```diff
@@ -91,7 +87,14 @@ func (c *reverseDNSCache) Add(translation *translation, now time.Time) bool {
 	return true
 }
 
-func (c *reverseDNSCache) Get(conns []ConnectionStats, now time.Time) map[util.Address][]string {
+func (c *reverseDNSCache) Get(conns []ConnectionStats) map[util.Address][]string {
+	c.mux.Lock()
+	defer c.mux.Unlock()
+
+	for _, val := range c.data {
+		val.inUse = false
```

Comment:
Would you please add a note perhaps here or at the type definition explaining the lifecycle of `inUse`.  I don't quite follow how it works.

Reply:
Note added.